deploy_tools/compile.py

```diff
@@ -89,7 +91,7 @@ def compile_project(
     optimize=True,
     optimize_runs=500,
     only_abi=False,
-    evm_version: str = "petersburg",
+    evm_version: str = None,
```

Comment:
Please use the default value here directly instead of relying on a magic value(None)
`evm_version = DEFAULT_EVM_VERSION`